TAKING ME
[{u'calltype': u'amd64', u'realsz': 394, u'diff': u'NEW', u'name': u'sym.getint', u'cc': 8, u'indegree': 0, u'nargs': 0, u'difftype': u'new', u'edges': 21, u'outdegree': 11, u'nlocals': 17, u'cost': 138, u'datarefs': [], u'callrefs': [{u'type': u'C', u'addr': 5043872, u'at': 4250482}, {u'type': u'C', u'addr': 5043872, u'at': 4250523}, {u'type': u'C', u'addr': 5043872, u'at': 4250564}, {u'type': u'C', u'addr': 4250768, u'at': 4250587}, {u'type': u'C', u'addr': 4250912, u'at': 4250759}, {u'type': u'C', u'addr': 4249168, u'at': 4250796}, {u'type': u'C', u'addr': 4250960, u'at': 4250614}, {u'type': u'C', u'addr': 4250848, u'at': 4250628}, {u'type': u'C', u'addr': 4249168, u'at': 4250694}, {u'type': u'C', u'addr': 4251040, u'at': 4250742}, {u'type': u'C', u'addr': 4251088, u'at': 4250727}, {u'type': u'J', u'addr': 4250747, u'at': 4250732}, {u'type': u'J', u'addr': 4250625, u'at': 4250598}], u'offset': 4250432, u'ebbs': 2, u'nbbs': 15, u'type': u'sym', u'size': 402}]
ERR. Target function sym.getint0 does not match sym.getint
Failed to patch all expected patches: 207  total patched: 5103
